apps/root/src/pages/earn/components/all-strategies-table-toolbar/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useThemeMode } from '@state/config/hooks';
+import { FormattedMessage, defineMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { ContainerBox, InputAdornment, SearchIcon, TextField, Typography, colors } from 'ui-library';
+
+const StyledSearchBox = styled(TextField)`
```

Comment:
Just a nit on this one: I think it would be better to make the textfield fullWidth but set it in the container and then we modify the containers size.

Not necessary tho